src/pyhf/constraints.py

```diff
@@ -58,10 +59,11 @@ def __init__(self, pdfconfig, batch_size=None):
             )
 
             _normal_sigmas = default_backend.concatenate(normal_constraint_sigmas)
-            sigmas = default_backend.reshape(_normal_sigmas, (1, -1))  # (1, normals)
-            self._batched_sigmas = default_backend.tile(
-                sigmas, (self.batch_size or 1, 1)
-            )
+            if self.batch_size:
+                sigmas = default_backend.reshape(_normal_sigmas, (1, -1))
+                self._sigmas = default_backend.tile(sigmas, (self.batch_size or 1, 1))
```

Comment:
should just be `self.batch_size` since you know it's not `None`

Reply:
done